src/components/Explore/Explore.tsx

```diff
@@ -106,6 +120,28 @@ const Explore: React.FunctionComponent<{
     setSelectedLocations(changedLocations);
   };
 
+  const exploreRef = useRef<HTMLDivElement>(null);
```

Comment:
I reworked the way we make /explore/ URLs auto-scroll to the explore component so that it matches the way compare works.  The way we did it before in ChartsHolder couldn't be used anymore because it was based on chartId instead of the URL.

Also, this should automatically work if we end up adding explore to the homepage.